site/content/blog/2021-08-01-whats-new-in-svelte-august-2021.md

```diff
@@ -0,0 +1,65 @@
+---
+title: What's new in Svelte: August 2021
+description: Shadow DOM, export and await - oh my!
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+From The Changelog ([JS Party Ep. 182](https://changelog.com/jsparty/182)) to Svelte Radio (Episodes [29](https://share.transistor.fm/s/adc23e84) and [30](https://share.transistor.fm/s/6316622d)), it seems that folks couldn't help but talk about Svelte, this month! Also, shadow DOM support and new export and await functionality are new in Svelte.
+
+## New in Svelte
```

Comment:
July was the most active month for the Svelte core repo since late 2019 as we really worked to reduce the number of outstanding PRs and saw the release of Svelte 3.39.0, 3.40.0, and 3.41.0. Tons of bug fixes were added as well as the following new features:

* Add `|trusted` event modifier (#6137)
* Made new `svelte/ssr` package to support work on improving SvelteKit SSR (#6416)
* Added new `errorMode` compiler option to support improved preprocessing of TypeScript files (#6194)

Reply:
Added